migrate-contexts/README.md

```diff
@@ -0,0 +1,72 @@
+# Migrating Contexts from CircleCI On-Prem to CircleCI SaaS
+
+## Extract Contexts from On-Prem
+
+### Get List of Contexts
+1. Get onto the `services` box
+2. Run the following to output a csv file of `contextname,context_uuid`
+```
+sudo docker exec postgres psql -U circle -d contexts_service_production -c "\copy (select name,id from contexts) TO STDOUT WITH CSV" > contexts.csv
+```
+
+### Get Contexts from REPL
+1. Get onto the `services` box
```

Comment:
Since these steps depend on the above, rather than `Get onto the services box` I think it's more clear to say
```
Complete the steps for `Get List of Contexts`
```